src/include/dhcp-server/widgets.rb

```diff
@@ -916,6 +838,39 @@ def AllSettingsButtonHandle(key, event)
       :main
     end
 
+    # Handle function for the 'Apply' button
+    def handle_apply(_key, event)
+      event_id = event["ID"]
+      if event_id == "apply"
```

Comment:
But the method is also called for `_cwm_wakeup`. In that call `key` is "apply" and `event["ID"]` is `_cwm_wakeup`. So it looks I cannot trust `key`... or maybe I'm misunderstanding something.